File: src/main/java/com/cloudcraftgaming/internal/calendar/event/EventCreatorResponse.java
package com.cloudcraftgaming.internal.calendar.event;

import com.google.api.services.calendar.model.Event;

/**
 * Created by Nova Fox on 1/4/2017.
 * Website: www.cloudcraftgaming.com
 * For Project: DisCal
 */
public class EventCreatorResponse {
    private final Boolean successful;

    private Event event;

    public EventCreatorResponse(Boolean _successful) {
        successful = _successful;
    }

    public EventCreatorResponse(Boolean _successful, Event _event) {
        successful = _successful;
        event = _event;
    }

    //Getters
    public Boolean isSuccessful() {
        return successful;
    }

    public Event getEvent() {
        return event;
    }

    //Setters
    public void setEvent(Event _event) {
        event = _event;
    }
}

File: core/src/main/kotlin/org/dreamexposure/discal/core/object/event/EventCreatorResponse.kt
package org.dreamexposure.discal.core.`object`.event

import com.google.api.services.calendar.model.Event
import discord4j.core.`object`.entity.Message

data class EventCreatorResponse(
        val successful: Boolean,
        val event: Event?,
        val creatorMessage: Message?,
        val edited: Boolean,
)
